frontend/app/src/components/history/events/HistoryEventNote.vue

```diff
@@ -53,9 +53,21 @@ function isLinkType(t: any): t is keyof ExplorerUrls {
     v-bind="$attrs"
     class="inline-flex items-center gap-x-1 flex-wrap"
   >
-    <template v-for="(note, index) in formattedNotes">
+    <template
+      v-for="(note, index) in formattedNotes"
+      :key="index"
+    >
+      <template v-if="note.type === 'word' && note.word">
```

Comment:
I think it's better to make this another NoteType (NoteType.FLAG)

Reply:
and also you have to make exception, to render this only if counterparty is gnosis pay